addons/kommander/1.1.x/kommander.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: kommander
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-4"
-    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-5"
```

Comment:
I left this `1.1.0-` here since I'm not sure how kubeaddons is parsing it. ideally It should have been `1.1.0-beta.1-1`, but I'm not sure if thats going to be handled in a weird way. 🤔

Reply:
It has to be valid semver is the only rule, as the value is sorted using native semver sorting.

Reply:
yeah, but that would mean `1.1.0-4` is "newer" than `1.1.0-beta.1-5`, correct? 🤔